portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,7 +34,38 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Jacob!</h1>");
+    Query query = new Query("Task");
```

Comment:
Good practices is to put table, column name into constant to prevent typos. 

private final static String tableName = "Task";